src/System.Security.Permissions/tests/CodeGroupTests.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Security.Permissions.Tests
+{
+    public class CodeGroupTests
+    {
+        [Fact]
+        public static void FileCodeGroupCallMethods()
+        {
+            Policy.FileCodeGroup fcg = new Policy.FileCodeGroup(new Policy.GacMembershipCondition(), new FileIOPermissionAccess());
+            Policy.CodeGroup cg = fcg.Copy();
+            bool equals = fcg.Equals(new object());
+            int hash = fcg.GetHashCode();
+            Policy.PolicyStatement ps = fcg.Resolve(new Policy.Evidence());
+            cg = fcg.ResolveMatchingCodeGroups(new Policy.Evidence());
```

Comment:
Here and elsewhere in the tests, is there anything we want to Assert?  Or we're just verifying that nothing throws/crashes when invoked?